### DISCUSSION TITLE
Utils.py ignored if DC feed is enabled

### DISCUSSION THREAD

**Discussion by stefan1967** [Louisvdw/dbus-serialbattery#452] [General]
Hello,

unfortunately is seems that all parameters from utils.py are ignored as long as DC-feed is enabeled.
The driver is working, because the soc parameters are visible in the GUI (Parameters CCL), but the victron system ignors all settigs (No SOC based current reduction, no cell voltage reduction).

Is there a solution or workaround available, seems this is a victron general problem.

Thanks a lot!
Stefan

**Comment by mr-manuel** [comment]
Hi, did you try to ask this in the Victron Community? As you sad, this is probably a Victron related problem.
